src/mem/Pool.h

```diff
@@ -166,9 +166,14 @@ class MemPools
     void setDefaultPoolChunking(bool const &);
 
     MemImplementingAllocator *pools = nullptr;
-    ssize_t mem_idle_limit = (2 << 20) /* 2MB */;
     int poolCount = 0;
     bool defaultIsChunked = false;
+
+private:
+    /// Limit for allocated (but unused) memory in memory pools.
```

Comment:
If we are going to polish all this, let's clarify that this limit is about bytes rather than object count:

```suggestion
    /// Limits the cumulative size of allocated (but unused) memory in all pools.
```